doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1791 @@
+L<EVP_PKEY-RSA(7)>, L<EVP_PKEY-EC(7)> and L<EVP_PKEY-X25519(7)>.
+The 'quick' one-shot function L<EVP_PKEY_Q_keygen(3)> and macros for the most
+common cases: <EVP_RSA_gen(3)> and L<EVP_EC_gen(3)> may also be used.
+
+=head4 Deprecated low-level key reading and writing functions
+
+Use of low-level objects (such as DSA) has been informally discouraged from use
+for a long time. Functions to read and write these low-level objects (such as
+PEM_read_DSA_PUBKEY()) should be replaced. Applications should instead use
+L<OSSL_ENCODER_to_bio(3)> and L<OSSL_DECODER_from_bio(3)>.
+
+=head4 Deprecated low-level key printing functions
+
+Use of low-level objects (such as DSA) has been informally discouraged from use
+for a long time. Functions to print these low-level objects such as
+DSA_print() should be replaced with the equivalent EVP_PKEY functions.
+Application should use one of L<EVP_PKEY_print_public(3)>,
+L<EVP_PKEY_print_private(3)>, L<EVP_PKEY_print_params(3)>,
+L<EVP_PKEY_print_public_fp(3)>, L<EVP_PKEY_print_private_fp(3)> or
+L<EVP_PKEY_print_params_fp(3)>. Note that internally these use
+L<OSSL_ENCODER_to_bio(3)> and L<OSSL_DECODER_from_bio(3)>.
+
+=head3 Deprecated function mappings
+
+The following functions have been deprecated in 3.0.
+
+=over 4
+
+=item AES_bi_ige_encrypt and AES_ige_encrypt
+
+There is no replacement for the IGE functions. New code should not use these modes.
+These undocumented functions were never integrated into the EVP layer.
+They implemented the AES Infinite Garble Extension (IGE) mode and AES
+Bi-directional IGE mode. These modes were never formally standardised and
+usage of these functions is believed to be very small. In particular
+AES_bi_ige_encrypt() has a known bug. It accepts 2 AES keys, but only one
+is ever used. The security implications are believed to be minimal, but
+this issue was never fixed for backwards compatibility reasons. 
+
+=item AES_encrypt, AES_decrypt, AES_set_encrypt_key, AES_set_decrypt_key,
+AES_cbc_encrypt, AES_cfb128_encrypt, AES_cfb1_encrypt, AES_cfb8_encrypt,
+AES_ecb_encrypt and AES_ofb128_encrypt
+
+=item AES_unwrap_key, AES_wrap_key
+
+See L</Deprecated low-level encryption functions>
+
+=item AES_options
+
+There is no replacement. It returned a string indicating if the AES code was unrolled.
+
+=item ASN1_digest, ASN1_sign and ASN1_verify
+
+There is no replacement. These old functions are not used, and could be disabled
+with the macro NO_ASN1_OLD since OpenSSL 0.9.7.
+
+=item ASN1_STRING_length_set
+
+Use L<ASN1_STRING_set(3)> or L<ASN1_STRING_set0(3)> instead.
+This was a potentially unsafe function that could change the bounds of a
+previously passed in pointer.
+
+=item BF_encrypt, BF_decrypt, BF_set_key, BF_cbc_encrypt, BF_cfb64_encrypt,
+BF_ecb_encrypt and BF_ofb64_encrypt
+
+See L</Deprecated low-level encryption functions>.
+The Blowfish algorithm has been moved to the L<Legacy Provider|/Legacy Algorithms>.
```

Comment:
Should be a reference to EVP_EncryptInit etc here and for similar encryption deprecations below

Reply:
It is all described in the link L</Deprecated low-level encryption functions>